mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -459,6 +462,8 @@ impl NexusChild {
             state: AtomicCell::new(ChildState::Init),
             prev_state: AtomicCell::new(ChildState::Init),
             remove_channel: mpsc::channel(0),
+            guid: Guid::from(uuid::Uuid::nil()),
```

Comment:
Is this nil "here" because it will be overridden later? If so should this be an Option?

Reply:
ok although I'd suggest adding some kind of test or assert in the code that makes sure that we do set the GUID.